docs/dev/coding_standards.rst

```diff
@@ -20,8 +20,9 @@ Many editors/IDEs can run this tool in the background while you work, showing re
 .. _flake8: https://flake8.pycqa.org/en/latest/
 
 It is allowed but not required to use the `black`_ code formatter. 
-To avoid introducing unrelated changes when working on an existing file, consider using the `darker`_ tool.
-It is compatible with :code:`black`, but only formats regions that show as changed in Git.
+To avoid introducing unrelated changes when working on an existing file, it is recommended to use the `darker`_ tool.
+This helps keep the focus on the implementation instead of unrelated formatting, and helps reviewers reviewing the code.
```

Comment:
```suggestion
This helps to keep the focus on the implementation instead of unrelated formatting, and thereby facilitates code reviews.
```
I did not like two times review in a row.

Thanks for tackling that issue.